### PR TITLE
[Core Commands] Add Help settings view

### LINKED ISSUE #4013: [Help] Add current settings to `[p]helpset` cmd
# Feature request

#### Select the type of feature you are requesting:

- [ ] Cog
- [x] Command
- [ ] API functionality

#### Describe your requested feature

Add current settings display (as in `[p]modset`/`[p]set`) for `[p]helpset` command

### PR DESCRIPTION
**Pull request by mikeshardmind**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [X] New feature

### Description of the changes

Adds a command for viewing the current help settings.

resolves #4013 depends on #4011

### CODE REVIEW

**Comment by mikeshardmind** [comment]
Just for a quick reference of how this looks with the default formatter, though there's a small bit about prepending a warning that these settings may not be used if a non-default formatter is in use.
image

**Comment by Jackenmen** [comment]
Maybe this would look better in code block?

**Comment by mikeshardmind** [comment]
I'm not convinced that it looks better [in a code block], especially when a custom tagline is in use (which I realize isn't shown in the quick example)

**Comment by Drapersniper** [comment]
Since the tagline is only added as `"{tagline_info}"` couldnt we add the first art into a code block followed by the tagline.
IMO it will kee in line with the existing Setting dislays in core.

**Comment by Jackenmen** [comment]
Needs conflict resolution

File: redbot/core/commands/help.py [hunk 1]
```diff
@@ -94,6 +94,40 @@ async def from_context(cls, context: Context):
...
+                return _("Yes")
+            return val
+
+        data = {k: bool_transformer(v) for k, v in dc_asdict(self).items()}
+
+        if not self.delete_delay:
+            data["delete_delay"] = _("Disabled")
+
+        if tag := data.pop("tagline", ""):
+            tagline_info = _("\nCustom Tagline: {tag}").format(tag)
```
Comment (Jackenmen) [inline redbot/core/commands/help.py]:
```suggestion
            tagline_info = _("\nCustom Tagline: {tag}").format(tag=tag)
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]
It would be nice if `Delete delay` were suffixed with " seconds" when it's set.

If you won't be able to do this, I'll just do a follow up PR for it later after merging

**Review by Jackenmen — APPROVED** [review]
Looks good to me, thanks for the follow up!

**Pull request merged**